src/Catty/DataModel/UserVariable/UserVariable.swift

```diff
@@ -21,7 +21,7 @@
  */
 
 @objc(UserVariable)
-@objcMembers class UserVariable: NSObject, CBMutableCopying {
+@objcMembers class UserVariable: NSObject, CBMutableCopying, UserDataProtocol {
```

Comment:
One more thing: I think we can remove the _CBMutableCopying_ protocol here, since it is already implemented in the _UserDataProtocol_.